department-of-veterans-affairs/caseflow-efolder
# Implement fetch document content api endpoint

connects #504 

Steps to validate:
1. Test eFolder Express that it functions as normal
2. Test API call /api/v1/documents/document_id:
    a) enable the feature `reader_api`
    b) document_id is the DB ID of the document that should already exist in the EX DB
    c) example using CURL: `curl -i -X GET http://localhost:3000/api/v1/documents/23548`
    d) disable the feature flag as it is the only thing that protects the API (no authentication in place)
